tests/structuredheadings/setnumberingstyletolist.js

```diff
@@ -132,6 +132,33 @@
         afterHtml: afterHtmlWithSelection
       };
       this.assertComboBeforeAfter(opts);
+    },
+    "applying a simple style deeply": function() {
+      var comboItem = "List: A. B. C.";
+      var beforeHtmlWithSelection = "<ol class=\"list-upper-alpha\"><li>foo" +
+        "<ol><li>^bar</li></ol></li></ol>";
+      var afterHtmlWithSelection = "<ol class=\"list-upper-alpha\"><li>foo" +
+        "<ol class=\"list-upper-alpha\"><li>^bar</li></ol></li></ol>";
```

Comment:
writing nested lists by hand is terrible, since the nested lists are actually a sublist within the first list item